app/services/replacement_parameters_service.rb

```diff
@@ -1,11 +1,12 @@
 class ReplacementParametersService
-  attr_accessor :body, :client, :preparer_user, :locale, :tax_return
+  attr_accessor :body, :client, :preparer_user, :locale, :tax_return, :intake
```

Comment:
This is relevant to this PR because we've got to send the "right" login link based on the type of intake you have. the <<Client.LoginLink>> needs to get replaced with one thing if it's being sent to a GYR intaker and another thing if its a CTC intaker. Otherwise, when they click the login link they'd always end up on GYR.